files/opi/boot.cmd

```diff
@@ -0,0 +1,6 @@
+setenv bootargs earlyprintk console=tty0 console=ttyAMA0 root=/dev/mmcblk0p2 rootfstype=ext4 rw rootwait init=/bin/systemd
```

Comment:
@eugulixes do we need earlyprintk? I suppose it was needed for debugging?

Reply:
It shows the messages from the earliest phases of the Linux boot process. Some of the images already use the option. We have to either keep it in all the images or get rid of it from all the images.